core/src/main/java/org/elasticsearch/cluster/routing/allocation/decider/EnableAllocationDecider.java

```diff
@@ -98,7 +98,7 @@ public void setEnableAllocation(Allocation enableAllocation) {
     @Override
     public Decision canAllocate(ShardRouting shardRouting, RoutingNode node, RoutingAllocation allocation) {
         if (allocation.ignoreDisable()) {
-            return allocation.decision(Decision.YES, NAME, "allocation is explicitly ignoring any disabling of allocation");
+            return allocation.decision(Decision.YES, NAME, "explicitly ignoring any disabling of allocation due to manual _reroute call");
```

Comment:
The `ignoreDisable` flag is only set when the commands of the _reroute call are executed, not when the actual reroute happens:

https://github.com/elastic/elasticsearch/blob/master/core/src/main/java/org/elasticsearch/cluster/routing/allocation/AllocationService.java#L259

What I wanted to say with the above is that I think the message should reflect that: It's just ignored for the commands.